### PR TITLE
add services section

### PR DESCRIPTION
**Pull request by 1Alisa1**

### CODE REVIEW

File: src/components/cardService/cardService.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import styles from './cardService.module.scss';
+
+interface CardProps {
+  title: string;
+  description: React.ReactNode;
+  handleClick: () => void;
```
Comment (N0rator) [inline src/components/cardService/cardService.tsx:6]: `HandleClick` name means that eventlistener assigned on whole card, which is not true.
I suggest to use more semanitically correct name, for example `onButtonClick`.

File: src/components/cardService/cardService.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+const CardService: React.FC<CardProps> = ({ title, description, handleClick }) => {
```
Comment (N0rator) [inline src/components/cardService/cardService.tsx:9]: `CardService` is little weird name for component, services is a different entity in programming. I suggest, to rename it somehow.

File: src/components/cardService/cardService.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import styles from './cardService.module.scss';
+
+interface CardProps {
```
Comment (N0rator) [inline src/components/cardService/cardService.tsx:3]: I suggest to rename it, to fit pattern: {component name}Props.

**Review by N0rator — CHANGES_REQUESTED** [review]

File: src/components/cardService/cardService.tsx (hunk 3 above)
Comment (1Alisa1) [reply src/components/cardService/cardService.tsx]: fixed

File: src/components/cardService/cardService.tsx (hunk 2 above)
Comment (1Alisa1) [reply src/components/cardService/cardService.tsx]: fixed

File: src/components/cardService/cardService.tsx (hunk 1 above)
Comment (1Alisa1) [reply src/components/cardService/cardService.tsx]: fixed

**Pull request merged**